yt/utilities/lib/platform_dep.h

```diff
@@ -1,6 +1,7 @@
 #include <math.h>
 #ifdef MS_WIN32
 #include "malloc.h"
+#include <float.h>

```

Comment:
`_fpclass`, `_FPCLASS_NN` and `_FPCLASS_PN` are Windows only symbols defined in `<float.h>`. It seems that CPython 3.10 and older leak `<float.h>`, but CPython 3.11 doesn't, hence the fix.